src/api/app/views/webui/attribute/_attributes.html.haml

```diff
@@ -14,7 +14,10 @@
           %td{ class: "#{attribute.namespace}-#{attribute.name}" }
             = attribute.fullname
           %td{ class: "#{attribute.namespace}-#{attribute.name}-values" }
-            = attribute.values.map(&:value).join(', ')
+            %ul
+            - attribute.values.each do |v|
+              %li
```

Comment:
The bullet point looks IMO really strange... what was the problem of the comma? can we see an screenshot of a multivalue case? The one you put have a single value, doesn't they? :confused:

Reply:
Actually: do you have any real life example where we use multiple values for an attribute?

Reply:
@coolo 

> Actually: do you have any real life example where we use multiple values for an attribute?

no, and if it doesn't exit, that would be another reason to do this differently. :thinking: